client/tests/integration/offline_peers.rs

```diff
@@ -13,11 +15,21 @@ fn genesis_block_is_commited_with_some_offline_peers() {
     let (network, iroha_client) = rt.block_on(<Network>::start_test_with_offline_and_set_n_shifts(
         4,
         1,
-        1,
         Some(10_560),
     ));
     wait_for_genesis_committed(&network.clients(), 1);
 
+    let parameter = Parameter::new(
+        ParameterId {
+            name: Name::from_str("MaxTransactionsInBlock").expect("Failed to parse Name"),
+        },
+        1u32.to_value(),
+    );
+    let param_box = SetParameterBox::new(parameter);
+    iroha_client
+        .submit_blocking(param_box)
+        .expect("Failed to submit transaction");
```

Comment:
Yeah, we should do something like this. Initially i don't want to be smart about integration tests if we want to deprecate them.
But if we define this in client it will have more broad applications than just in tests.

Reply:
I've added `ParametersBuilder`, what do you think?